gmusicapi/clients/musicmanager.py

```diff
@@ -1,17 +1,19 @@
 from __future__ import print_function, division, absolute_import, unicode_literals
-from future import standard_library
-standard_library.install_aliases()
+from future.utils import PY3
 from past.builtins import basestring
 from builtins import *  # noqa
+
 import os
 from socket import gethostname
 import time
-import urllib.request
-import urllib.parse
-import urllib.error
 from uuid import getnode as getmac
 import webbrowser
 
+if PY3:
```

Comment:
My understanding was that if you are supporting Python 2 for a long time and Python 4 comes along, you don't have to scramble to update the conditional.

In the vein of treating Python 2 like the exceptional case and not Python 3.

But if you can decide that before Python 4 comes along, the Python 2 support will be good and dead, it doesn't matter.